Modules/Setup.stdlib.in

```diff
@@ -37,6 +37,13 @@
 # with ./configure --with-system-libmpdec
 @MODULE__DECIMAL_TRUE@_decimal _decimal/_decimal.c
 
+# hashing builtins, can be disabled with --without-builtin-hashlib-hashes
+@MODULE__MD5_TRUE@_md5 md5module.c
+@MODULE__SHA1_TRUE@_sha1 sha1module.c
+@MODULE__SHA256_TRUE@_sha256 sha256module.c
+@MODULE__SHA512_TRUE@_sha512 sha512module.c
+@MODULE__SHA3_TRUE@_sha3 _sha3/sha3module.c
+@MODULE__BLAKE2_TRUE@_blake2 _blake2/blake2module.c _blake2/blake2b_impl.c _blake2/blake2s_impl.c
```

Comment:
Also remove the "old" entries in `Modules/Setup`?

Reply:
``Modules/Setup.stdlib`` is not used yet. I like to get all modules into the new setup file and then we can decide what do do about the old setup file.